infrastructure/server/subdomain-flag.www.sub.any.js

```diff
@@ -0,0 +1,6 @@
+// META: global=window,dedicatedworker,sharedworker,serviceworker
```

Comment:
Does running in multiple globals test something interesting?

Reply:
(Particularly, SharedWorker is not available on Safari, so if you want to test on that we'll need to add an expectation file in `infrastructure/metadata/infrastructure/server/subdomain-flag.www.sub.any.js.ini` which allows the test to fail on Safari).

Reply:
No, not really (just cargo-culting here). I'll change this to `window` and that should avoid Safari being sad about `sharedworker`.

Reply:
> (Particularly, SharedWorker is not available on Safari, so if you want to test on that we'll need to add an expectation file in `infrastructure/metadata/infrastructure/server/subdomain-flag.www.sub.any.js.ini` which allows the test to fail on Safari).

Thanks, this is useful info!